README.md

```diff
@@ -1,63 +1,99 @@
+- Provides sophisticated domain and quality classification
+- Flexible interface for extending to your own classifier network
+
+[Personal identifiable information (PII) redaction](docs/user-guide/PersonalIdentifiableInformationIdentificationAndRemoval.rst)
+
+- Identification tools for removing addresses, credit card numbers, social security numbers, and more
+
+These modules offer flexibility and permit reordering, with only a few exceptions. In addition, the [NeMo Framework Launcher](https://github.com/NVIDIA/NeMo-Megatron-Launcher) provides pre-built pipelines that can serve as a foundation for your customization use cases.
+
+## Resources
+
 - [Documentation](docs/)
 - [Examples](examples/)
-- [Module Ablation and Compute Performance](#module-ablation-and-compute-performance)
+- [Tutorials](tutorials/)
 
-## Installation
+## Get Started
 
-NeMo Curator currently requires Python 3.10 and the GPU accelerated modules require CUDA 12 or above installed in order to be used.
+This section explains how to install NeMo Curator and use the Python library, Python modules, and CLI scripts. It also includes a list of tutorials to help you get started right away. Finally, this section explains how to use the NeMo Framework Launcher as an alternative method for interfacing with NeMo Curator.
 
-NeMo Curator can be installed manually by cloning the repository and installing as follows -
+## Requirements
 
-For CPU only modules:
-```
-pip install .
-```
+Before installing NeMo Curator, ensure that the following requirements are met:
 
-For CPU + CUDA accelerated modules
-```
-pip install --extra-index-url https://pypi.nvidia.com ".[cuda12x]"
-```
+- Python 3.10 (or above)
+- CUDA 12 (or above)
+- NVIDIA GPU (optional)
+
+## Install NeMo Curator
+
+Two options are available for installing NeMo Curator. You  can install it from the repository or through the NeMo Framework container.
+
+### Install from the Repository
+
+1. Clone the NeMo Curator repository in GitHub.
+
+2. Install the modules that you need.
+
+    To install the CPU-only modules:
+
+    ```
+    pip install
+    ```
+
+    To install the CPU and CUDA-accelerated modules:
+
+    ```
+    pip install --extra-index-url https://pypi.nvidia.com ".[cuda12x]"
+    ```
 
-### NeMo Framework Container
+### Install from the NeMo Framework Container
 
-NeMo Curator is available in the [NeMo Framework Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo). The NeMo Framework Container provides an end-to-end platform for development of custom generative AI models anywhere. The latest release of NeMo Curator comes preinstalled in the container.
+NeMo Curator is available in the [NeMo Framework Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo/tags). The latest release of NeMo Curator comes preinstalled in the container.
 
-## Usage
+## Use the Python Library
 
-### Python Library
+To download your dataset, build your pipeline, and curate your dataset:
```

Comment:
This doesn't read well. Could you please modify so it says something along the following lines:

The snippet below demonstrates the creation of a small data curation pipeline that downloads and curates a small subset of the Common Crawl dataset:

(the snippet goes here)

Reply:
Added sentence.